### ISSUE TITLE
[Samples] Need to update old samples

### ISSUE THREAD

**Issue by paulcam206** [microsoft/AdaptiveCards#4552]
# Details

Now that 1.3 has shipped, we have input labels and input validation. Scenario sample cards created pre-1.3 should have new 1.3 versions created that take advantage of both features. 

The pages showing pre-1.3 samples will *also* be updated with a banner that explains that the newer features should be used in order to light up accessibility. (see #4807)

## Cards to update
I *think* this is the list of items that need action. I searched for `Inputs.` under the samples directory and stripped out 1.3 items, items that already had a newer version, and items under `Tests`.
* [ ] Templates/Scenarios/ActivityUpdate.template.json
* [ ] Templates/Scenarios/CalendarReminder.template.json
* [ ] Templates/Scenarios/ExpenseReport.template.json
* [X] Templates/Scenarios/FoodOrder.template.json (`FoodOrderWithValidation.template.json`)
* [ ] Templates/Scenarios/FoodOrderWithValidation.template.json
* [ ] Templates/Scenarios/InputForm.template.json
* [X] Templates/Scenarios/Inputs.template.json (`InputsWithValidation.template.json`)
* [ ] v1.0/Elements/Action.Submit.json
* [ ] v1.0/Elements/AdaptiveCard.json
* [ ] v1.0/Elements/Image.ImageBaseUrl.json
* [ ] v1.0/Elements/Input.ChoiceSet.json
* [ ] v1.0/Elements/Input.Date.json
* [ ] v1.0/Elements/Input.Number.json
* [ ] v1.0/Elements/Input.Text.Style.json
* [ ] v1.0/Elements/Input.Text.json
* [ ] v1.0/Elements/Input.Time.json
* [ ] v1.0/Elements/Input.Toggle.json
* [ ] v1.0/Scenarios/ActivityUpdate.json
* [ ] v1.0/Scenarios/CalendarReminder.json
* [X] v1.0/Scenarios/FoodOrder.json
* [ ] v1.0/Scenarios/InputForm.json
* [X] v1.0/Scenarios/Inputs.json
* [ ] v1.2/Elements/Input.Text.InlineAction.json  
* [ ] v1.2/Scenarios/ExpenseReport.json

**Comment by paulcam206** [comment]
@matthidinger - care to wordsmith the banner in the description?

**Comment by RebeccaAnne** [comment]
A new version of Inputs.json was created with #4455. Currently here: [InputsWithValidation.json](https://raw.githubusercontent.com/microsoft/AdaptiveCards/main/samples/v1.3/Scenarios/InputsWithValidation.json).

**Comment by matthidinger** [comment]
How about something like this:

```
{
    "type": "Container",
    "verticalContentAlignment": "Center",
    "style": "warning",
    "items": [
        {
            "type": "TextBlock",
            "text": "**IMPORTANT:** This card is based on an older schema and doesn't make use of the **label** property for **Input** elements. Because of this, it doesn't properly support accessibility. [If your Host version is >= 1.3](https://aka.ms/ACHosts), please see [[NEW SAMPLE NAME]] for an updated version of this card.",
            "wrap": true
        }
    ]
}
```

**Comment by shalinijoshi19** [comment]
Assigning to you @almedina-ms per offline POA; @RebeccaAnne FYI. Pls update description if needed! thanks

**Comment by shalinijoshi19** [comment]
Outlining POA here per @RebeccaAnne's notes offline:
We propose to annotate samples on our website which use the old schema with a banner warning indicating that because they target an older version of the schema they are not accessible. 
In Q4, once more of our partners have picked up v1.3, we would remove the old samples with the warnings and replace them with accessible samples. #4612 tracks the latter work.

**Comment by RebeccaAnne** [comment]
Per discussion, we will be putting the banner warnings on the website itself, not the individual cards. Updated the description above to remove checkboxes for individual warning cards.

**Comment by almedina-ms** [comment]
assigning @jwoo-msft as he'll be working on accessibility fixes for iOS renderer
